### ISSUE TITLE
Add autocomplete field for long multi-select fields

### ISSUE THREAD

**Issue by dalepotter** [IATI/IATI-Query-Builder#34]
[User feedback](https://twitter.com/stevieflow/status/847383380383551488) suggests that the Select Reporting Organisation multi-select field on the [query builder](http://datastore.iatistandard.org/query/) is not very easy to use:

![image](https://cloud.githubusercontent.com/assets/8247168/24506314/f8c6d18a-1554-11e7-993c-c7a63e88f249.png)

Perhaps the approach used by d-Portal could be used? For example:

![image](https://cloud.githubusercontent.com/assets/8247168/24506278/dd35aa36-1554-11e7-9e4b-ae494fb367d7.png)

**Comment by YohannaLoucheur** [comment]
Agree, that would be a better interface. Would help especially when you're not sure of exact name but have the ID or some key words.
While we're at it, any way to make the window bigger (ie see more than 3 publishers)?
